src/project/qml/MuseScore/Project/internal/Properties/ProjectPropertiesFileInfoPanel.qml

```diff
@@ -78,7 +78,7 @@ ColumnLayout {
         spacing: 8
 
         PropertyItem {
-            propertyNameWidth: root.propertyNameWidth
+            propertyNameWidth: t_metrics.tightBoundingRect.width
```

Comment:
Isn't this change part of #17019?

Reply:
oops- you're totally right. Not sure how that ended up in this branch- still learning git branch mechanics. Thanks for letting me know. I'll try & see if I can get rid of that in this branch